src/Components/Header/Header.js

```diff
@@ -5,13 +5,13 @@ const Header = ({ showHome }) => {
     return (
         <header>
             <div>
-                <h4 className="site-title">TOMATILLOS</h4>
+                <h4 className="site-title">ROTTEN TOMATILLOS</h4>
```

Comment:
we should find a cool font or an image to put in there so that it looks more like a logo and less like boring text